insights/specs/__init__.py

```diff
@@ -471,7 +471,7 @@ class Specs(SpecSet):
     root_crontab = RegistryPoint()
     route = RegistryPoint()
     rpm_V_packages = RegistryPoint()
-    rsyslog_conf = RegistryPoint(filterable=True)
```

Comment:
@wushiqinlou you need to keep `filterable` here.  You can add `{}()` to a filter in the parser to ensure you get your opening and closing chars.

Reply:
@bfahr - I think we can return all the content of configuration file, and no need to add filter. If we keep filterable, and add filter for {}() in parser, we may miss necessary content like line 90, 91 in test_rsyslog_confs.py.

Reply:
@bfahr - Will add filterable as your suggestion.

Reply:
@bfahr - Updated as your suggestion, please have a check.